### PR TITLE
CONN-1161 & CONN-729

### PR DESCRIPTION
**Pull request by christophermay07**
Resolved more issues found during QA of CONN-1161 and CONN-729, and a bit more code cleanup.

### CODE REVIEW

File: Product/Production/Adapters/General/CONNECTDirectConfig/src/main/java/gov/hhs/fha/nhinc/directconfig/entity/Certificate.java [hunk 1]
```diff
@@ -308,7 +315,6 @@ public void clearData() {
         try {
             setData(NULL_CERT);
         } catch (CertificateException e) {
-            // TODO Auto-generated catch block
             e.printStackTrace();
```
Comment (jsmith2) [inline Product/Production/Adapters/General/CONNECTDirectConfig/src/main/java/gov/hhs/fha/nhinc/directconfig/entity/Certificate.java]: should be logged.

File: Product/Production/Adapters/General/CONNECTDirectConfig/src/main/java/gov/hhs/fha/nhinc/directconfig/processor/impl/DefaultBundleCacheUpdateProcessorImpl.java [hunk 2]
```diff
@@ -114,7 +113,7 @@ public void setRefreshProcessor(BundleRefreshProcessor refreshProcessor) {
      * {@inheritDoc}
      */
     @Override
-    @Handler
+    // @Handler
```
Comment (jsmith2) [inline Product/Production/Adapters/General/CONNECTDirectConfig/src/main/java/gov/hhs/fha/nhinc/directconfig/processor/impl/DefaultBundleCacheUpdateProcessorImpl.java:117]: delete this then.

File: Product/Production/Adapters/General/CONNECTDirectConfig/src/main/java/gov/hhs/fha/nhinc/directconfig/entity/Certificate.java (hunk 1 above)
Comment (christophermay07) [inline Product/Production/Adapters/General/CONNECTDirectConfig/src/main/java/gov/hhs/fha/nhinc/directconfig/entity/Certificate.java]: Great catch, not sure how that slipped past!  This is now being run through the logger.

File: Product/Production/Adapters/General/CONNECTDirectConfig/src/main/java/gov/hhs/fha/nhinc/directconfig/processor/impl/DefaultBundleCacheUpdateProcessorImpl.java (hunk 2 above)
Comment (christophermay07) [inline Product/Production/Adapters/General/CONNECTDirectConfig/src/main/java/gov/hhs/fha/nhinc/directconfig/processor/impl/DefaultBundleCacheUpdateProcessorImpl.java:117]: I added a comment to clarify why this has been left in (and also added back in the import, which had been auto-deleted on save).  Created CONN-1188 to track this issue.

**Pull request merged**